src/azure-cli-core/azure/cli/core/aaz/_field_value.py

```diff
@@ -326,8 +325,7 @@ def __init__(self, schema, data):
         self._len = 0
         if self._data is not None and self._data != AAZUndefined:
             for idx in self._data:
-                if idx + 1 > self._len:
-                    self._len = idx + 1
+                self._len = max(self._len, idx + 1)
```

Comment:
`src/azure-cli-core/azure/cli/core/aaz/_field_value.py:329:16: R1731: Consider using '_len = max(_len, idx + 1)' instead of unnecessary if block (consider-using-max-builtin)`